public/index.html

```diff
@@ -5,34 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
     <script src="%PUBLIC_URL%/outdated-browser-rework.min.js"></script>
-    <script>
-      outdatedBrowserRework({
-        browserSupport: {
-          Chrome: 67, // Includes Chrome for mobile devices
-          Edge: 40,
-          Safari: false,
-          "Mobile Safari": 10,
-          Opera: 54,
-          Firefox: 60,
-          Vivaldi: 1,
-          IE: false
-        },
-        isUnknownBrowserOK: true,
-        messages: {
-          en: {
-            outOfDate: "Terra may not function correctly in this browser.",
-            update: {
-              web: "If you experience issues, please try " + (!!window.chrome ? "updating" : "using") + " Google Chrome.",
-              googlePlay: "Please install Chrome from Google Play",
-              appStore: "Please update iOS from the Settings App"
-            },
-            url: "https://www.google.com/chrome/",
-            callToAction: (!!window.chrome ? "Update" : "Download") + " Chrome now",
-            close: "Close"
-          }
-        }
-      });
-    </script>
+    <script src="%PUBLIC_URL%/outdated-browser-message.js"></script>
```

Comment:
Tested by manually changing the acceptable version of Chrome in the production build and seeing the warning.